roc/log.go

```diff
@@ -147,6 +149,7 @@ func (standardLogger) Print(v ...interface{}) {
 }
 
 var (
+	logLevel   atomic.Value
```

Comment:
For consistency with other vars, let's rename it to `loggerLevel`.

There is no need to use atomic.Value here because it's just an int. You can use an int of an appropriate size instead, and corresponding atomic functions.